.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release to charmhub
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    uses: ./.github/workflows/ci.yaml
+
+  publish:
+    name: Release to CharmHub
+    needs:
+      - build
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Check libs
+        uses: canonical/charming-actions/check-libraries@2.1.0
```

Comment:
1) I recommend to bump here (and below) to version 2.2.2, there were 'libraries' specific refactoring.
2) I suggest to move check-libraries from release.yaml to [ci.yaml](https://github.com/canonical/mysql-operator/blob/main/.github/workflows/ci.yaml#L45), this will inform you about broken s3 library state:

> charmcraft fetch-lib 
>Library charms.data_platform_libs.v0.s3 has local changes, cannot be updated.

You can see example of PR libraries statuses check here: https://github.com/canonical/mysql-operator/pulls